client/src/hooks/useGetUser.tsx

```diff
@@ -0,0 +1,21 @@
+import { useEffect } from 'react';
+
+import api from 'src/api';
+
+import { useAuth } from './useAuth';
+
+export const useGetUser = (): void => {
+  const { signIn } = useAuth();
+
```

Comment:
Just 'useUser' + you need to make the hook return the value received from the request